### PR TITLE
First Pull Request: To Do List

### PR DESCRIPTION
**Pull request by MussieTeka**
This pull request adds web pack config, creates index.html with a To-Do List placeholder, and script.js with an array of tasks. It also renders the list on page load and applies CSS styling using style.css.

### CODE REVIEW

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @MussieTeka , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used Webpck ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/script.js [hunk 1]
```diff
@@ -1 +1,57 @@
...
+    const li = document.createElement('li');
+    li.innerHTML = `
+      <input type="checkbox" />
+      <span>${input.value}</span>
+      <i class="fa fa-ellipsis-v"></i>
+    `;
+    todoList.appendChild(li);
+    input.value = '';
+  }
+}
```
[the remark this answers is not in the document]
Comment (MussieTeka) [reply src/script.js:24]: Thank you for the feedback. I have made the necessary changes to include the array of simple to-do tasks with three keys as required. The updated code now meets all requirements. Thank you for pointing out the missed requirement.

File: src/script.js [hunk 2]
```diff
@@ -1 +1,57 @@
...
+    const item = items[i];
+    if (item.classList.contains('completed')) {
+      todoList.removeChild(item);
+    }
+  }
+}
+
+function refreshPage() {
+  window.location.reload();
+}
```
[the remark this answers is not in the document]
Comment (MussieTeka) [reply src/script.js:90]: Thanks for the feedback. I have added an event listener to the window object's "load" event to render the list of tasks on page load as required. The code now meets all requirements.

**Review by Ismailco — APPROVED** [review]
### Hi @MussieTeka, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️ Nice UI
✔️ No linter errors
✔️ Used GitHub Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**